CMS-Enterprise/easi-app
# [EASI-4653] Modify CEDAR system role emails

# EASI-4653

## Description

- This adds/modifies some links and content in the CEDAR system team member email, as well as modifies the trigger to only happen when a user is added.
- The resolver for roles and the core methods were untested, so I added tests as well.

## How to test this change

- Try adding roles to a user or removing them locally. With CEDAR and Okta mocking enabled it should send the correct emails without calling out to CEDAR.
- Run the tests
- Run the email script

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
